src/bin/version.rs

```diff
@@ -0,0 +1,91 @@
+use cjms::controllers::custodial::VERSION_FILE;
+use std::process::Command;
+use std::str;
+use std::{env, fs};
+
+fn main() -> std::io::Result<()> {
+    let (sha, tag) = match env::var("CI") {
+        Ok(_) => {
+            // If we're in CI use local variables
+            let sha = env::var("GITHUB_SHA").expect("Missing environment variable GITHUB_SHA");
+            let tag =
+                env::var("GITHUB_REF_NAME").expect("Missing environment variable GITHUB_REF_NAME");
+            (sha, tag)
+        }
+        Err(_) => {
```

Comment:
Ok, so this is what I think I was trying to get at in our earlier conversation but wasn't being articulate. Basically, my assumption is that every environment this app runs in will be using an image built from CI, with the exception of local. Which means that if my assumption is correct, the error block will only run on local environments. But do we care about local? Why not just return null if we can't find the right environment variables? This would simplify this a lot.

Actually, it would probably mean that you could obviate the need for this file generation system altogether and just pull from the env vars directly in the version controller.

Reply:
On guardian we also run on heroku which ends up being a different deploy situation. That said, I think you're right and we can probably burn this PR in the future. I'm going to add a clean-up subtask for this.